server/src/main/java/org/opensearch/cluster/service/MasterService.java

```diff
@@ -131,7 +132,13 @@ public class MasterService extends AbstractLifecycleComponent {
     protected final ClusterManagerTaskThrottler clusterManagerTaskThrottler;
     private final ClusterManagerThrottlingStats throttlingStats;
 
+    private final LongSupplier nanoTimeSupplier;
+
     public MasterService(Settings settings, ClusterSettings clusterSettings, ThreadPool threadPool) {
+        this(settings, clusterSettings, threadPool, System::nanoTime);
+    }
+
+    public MasterService(Settings settings, ClusterSettings clusterSettings, ThreadPool threadPool, LongSupplier timeProvider) {
```

Comment:
keep variable name same across classes - timeProvider/ timeSupplier

Reply:
Missed during final refactoring, will revise it in next commit.